spec/services/reject_application_by_default_spec.rb

```diff
@@ -28,4 +28,10 @@ def create_application
     described_class.new(application_choice: application_choice).call
     expect(application_choice.reload.rejected_by_default).to be true
   end
+
+  it 'sends a Slack notification' do
```

Comment:
Could we test the sending of the Slack message in these assertions? This is only testing that we schedule the job correctly (there are better ways of doing that: https://github.com/mperham/sidekiq/wiki/Testing).